launch_ros/launch_ros/actions/composable_node_container.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Module for the ComposableNodeContainer action."""
+
+from typing import List
+from typing import Optional
+
+from launch.action import Action
+from launch.launch_context import LaunchContext
+
+from .node import Node
+
+from ..descriptions import ComposableNode
+
+
+class ComposableNodeContainer(Node):
+    """Action that executes a container ROS node for composable ROS nodes."""
+
+    def __init__(
+        self,
+        *,
+        composable_nodes_description: Optional[List[ComposableNode]] = None,
```

Comment:
recommend `composable_node_descriptions` instead of `composable_nodes_description`

Reply:
Sure thing.